GCTC-NTGC/gc-digital-talent
# Feature - 3415 - My Applications Page

Resolves #3415 

## Summary

This creates the "My Applications" page, sorts them based off of their statuses, grouping by the associated colours and adding different action links based on their status.

## Screenshot

<img width="1175" alt="Screen Shot 2022-09-01 at 2 38 47 PM" src="https://user-images.githubusercontent.com/4127998/187988330-2303ce0b-1d6f-4505-8cd9-111525da03e3.png">

## Testing

For this, it is best to test in storybook since right now our seeder only creates a single application.

1. Run storybook `npm run storybook --workspace=talentsearch`
2. Navigate to the "My Applications" story
3. Confirm the colour links, colour and sorting aligns with the tables outlined in the connected issue (#3415)
